plugin.rb

```diff
@@ -340,11 +340,17 @@ def accepted_answer
   require_dependency 'topic_list_item_serializer'
 
   class ::TopicListItemSerializer
-    attributes :has_accepted_answer
+    attributes :has_accepted_answer, :can_have_answer
 
     def has_accepted_answer
       object.custom_fields["accepted_answer_post_id"] ? true : false
     end
+
+    def can_have_answer
```

Comment:
Agreed, should the same also be applied to the "has_accepted_answer" property (plugin.rb line 345)